### PR TITLE
Added logger e2e tests

### PR DESCRIPTION
**Pull request by liadlauber**

### CODE REVIEW

File: test/k8s_tests/logger_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+)
+
+const (
+	outputSuffix = "-output"
+	flowSuffix   = "-flow"
+	testIndex    = "test"
+)
+
+// createCappWithLogger creates a Capp instance with the specified logger type and returns the created Capp object.
+func createCappWithLogger(logType string) *v1alpha1.Capp {
```
Comment (fran2810) [inline test/k8s_tests/logger_e2e_test.go:19]: should be in utils

File: test/k8s_tests/logger_e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	case mock.ElasticType:
+		capp.Spec.LogSpec = mock.CreateElasticLogSpec()
+	case mock.SplunkType:
+		capp.Spec.LogSpec = mock.CreateSplunkLogSpec()
+	}
+	return utilst.CreateCapp(k8sClient, capp)
+}
+
+// createCredentialsSecret creates a Kubernetes secret containing credentials for the specified logger type.
+func createCredentialsSecret(logType string) {
```
Comment (fran2810) [inline test/k8s_tests/logger_e2e_test.go:31]: should be in utils

File: test/k8s_tests/mocks/base.go [hunk 3]
```diff
@@ -112,4 +128,67 @@ func CreateEnvVarObject(refName string) *[]corev1.EnvVar {
 			},
 		},
 	}
-}
\ No newline at end of file
+}
+
+func CreateElasticLogSpec() rcsv1alpha1.LogSpec {
```
Comment (fran2810) [inline test/k8s_tests/mocks/base.go:133]: make files for looger objects

File: test/k8s_tests/utils/logger_adapter.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+)
+
+// GetOutput fetch existing and return an instance of Output.
+func GetOutput(k8sClient client.Client, name string, namespace string) *loggingv1beta1.Output {
+	output := &loggingv1beta1.Output{}
+	GetResource(k8sClient, output, name, namespace)
+	return output
+}
+
+// GetFlow fetch existing and return an instance of Flow.
```
Comment (fran2810) [inline test/k8s_tests/utils/logger_adapter.go:15]: present simple

File: test/k8s_tests/utils/logger_adapter.go [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	loggingv1beta1 "github.com/kube-logging/logging-operator/pkg/sdk/logging/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// GetOutput fetch existing and return an instance of Output.
```
Comment (fran2810) [inline test/k8s_tests/utils/logger_adapter.go:8]: present simple

**Review by fran2810 — COMMENTED** [review]
The test looks very good but the wrappers theres more work to do.

**Review by fran2810 — APPROVED** [review]
/lgtm

**Review by mzeevi — APPROVED** [review]

**Pull request merged**